provider/openstack/firewaller.go

```diff
@@ -480,7 +688,6 @@ func (c *defaultFirewaller) closePortsInGroup(nameRegExp string, portRanges []ne
 		return err
 	}
 	neutronClient := c.environ.neutron()
-	// TODO: Hey look ma, it's quadratic
```

Comment:
quadratic doesn't matter when N is small, but to be fair this code handles the global firewalling group, so I'll reinstate :)